app/models/orchestration_template_cfn.rb

```diff
@@ -54,11 +54,18 @@ def self.eligible_manager_types
     @eligible_manager_types ||= [ManageIQ::Providers::Openstack::CloudManager]
   end
 
-  # return the parsing error message if not valid JSON; otherwise nil
+  # return the parsing error message if content is not parsable; otherwise nil
   def validate_format
+    self.format = "json"
```

Comment:
validate format based on `format` method instead of setting it to `json`, then we can properly return paring error.